### DISCUSSION TITLE
How to export and replay workflow in another environment for debugging

### DISCUSSION THREAD

**Discussion by balchua** [cschleiden/go-workflows#325] [Q&A]
I don't know if this is something anyone has tried, i wanted to export the history of a failed workflow and then import it to another environment to simulate the workflow.
Has anyone done something similar?

Thanks

**Comment by cschleiden** [comment]
That's currently not supported but I've thought about it. Shouldn't be too difficult to export from the diag UI and then either import into another environment or use the workflow tester to replay.
